_includes/mentee-guidelines.html

```diff
@@ -24,7 +24,8 @@ <h3>2. Help us to get to know you</h3>
   aspect of this process. Introduce yourself on our
   <a href="mailto: opensuse-project@opensuse.org">mailing list</a>
   ("Hey I'm thinking about becoming a mentee in 101 and I'm not
-  sure about XXX and have questions about YYY"), chat with us on
+  sure about XXX and have questions about YYY"), write as in
```

Comment:
I would use "get in touch with us on.. " or "Tweet us at @opensusementors", but in any case "write as in" sounds like instructions on how to write.

Reply:
_tweet us at @opensusementors_ sounds great!